stm32-modules/thermocycler-gen2/firmware/thermal/thermal_hardware.c

```diff
@@ -56,7 +56,7 @@
  * On the higher end (low-priority) because timing
```

Comment:
[nit] probably want to change this comment given that it's now on the lower end (high priority)